pkg/pool-manager/virtualmachine_pool.go

```diff
@@ -484,7 +477,7 @@ func (p *PoolManager) vmWaitingCleanupLook() {
 
 		configMapUpdateNeeded := false
 		if configMap.Data == nil {
-			logger.Info("the configMap is empty", "configMapName", names.WAITING_VMS_CONFIGMAP, "macPoolMap", p.macPoolMap)
+			logger.V(1).Info("the configMap is empty", "configMapName", names.WAITING_VMS_CONFIGMAP, "macPoolMap", p.macPoolMap)
```

Comment:
I see that in some places you use `logger.V(1).info` and in some like in row 455 you use just `logger.info`, whats the difference?

Reply:
`logger.V(1).info` will result to a `DEBUG` log, and thus will **not** appear when looking at logs on production (i.e. ,HCO/CNAO/CNV deploy)
`logger.info`, will appear as `INFO` and thus will appear there.